ceph/ceph/values.yaml

```diff
@@ -299,6 +299,13 @@ ceph_mgr_enabled_modules:
 #    num_rep: "3"
 #    min_size: "2"
 
+# You can run any ceph commands upon deployment
+# or upgrade by setting the following.
+# These will be executed via kubectl exec
```

Comment:
perhaps mention the command must not contain ';'

Reply:
done